service/src/main/java/com/epam/esm/dto/CertificateDTO.java

```diff
@@ -0,0 +1,49 @@
+package com.epam.esm.dto;
+
+import javax.validation.constraints.*;
```

Comment:
Please replace it with single import statements.

Reply:
done!